.github/workflows/CI.yml

```
@@ -24,14 +24,17 @@ concurrency:
   cancel-in-progress: ${{ startsWith(github.ref, 'refs/pull/') }}
 jobs:
   test:
-    name: Julia ${{ matrix.version }} - ${{ matrix.os }} - ${{ matrix.arch }} - ${{ github.event_name }}
+    name: Julia ${{ matrix.version }} - ${{ matrix.test_group }} - ${{ matrix.os }} - ${{ matrix.arch }} - ${{ github.event_name }}
```

lets change the order of this info so its sorted in a much relevant way. also the os, arch and github event name don't seem to give much information because they are always the same.

```suggestion
    name: ${{ matrix.test_group }} / Julia ${{ matrix.version }}
```